tests/sentry/api/endpoints/test_organization_integration_repository_project_path_configs.py

```diff
@@ -99,8 +101,82 @@ def test_basic_get(self):
             "defaultBranch": None,
         }
 
-    def test_basic_post(self):
-        response = self.make_post()
+    def test_basic_get_with_projectId(self):
+        path_config1 = RepositoryProjectPathConfig.objects.create(
```

Comment:
I made the `create_code_mapping` test util that could be helpful here, although realizing probs could  be updated to have the default branch be master too